pkgs/development/python-modules/python-hglib/default.nix

```
@@ -1,24 +1,28 @@
 {
   lib,
   buildPythonPackage,
-  fetchPypi,
+  fetchhg,
   mercurial,
-  nose,
 }:
 
 buildPythonPackage rec {
   pname = "python-hglib";
-  version = "2.6.2";
+  version = "2.6.2+nixpkgs.1";
```

according to https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md

```suggestion
  version = "2.6.2-unstable-2024-06-17";
```

---

Naive application of your suggestion causes:
```
setuptools.extern.packaging.version.InvalidVersion: Invalid version: '2.6.2-unstable-2024-06-17'
```

`setuptools` requires a valid version which `setup.py` can detect.
`.hg_archival.txt` or somewhere should include the valid version string `2.6.2+nixpkgs.1`, but it is different from a version placed in derivation. Is it ok?

---

I could not find the consensus (e.g. documentation or issues) on how to report _package revision_ but it seems like keep the version as is.

https://github.com/NixOS/nixpkgs/blob/39fb5bf8103700d85643a9733fb4156804a59196/pkgs/development/python-modules/tubes/default.nix#L13


So in this case, remove temporary versioning? Not sure.

---

Applied your suggestion.